tests/Tests/100__deploy/100__installation/109__operator.robot

```diff
@@ -24,6 +24,19 @@ Verify Odh-deployer Checks Cluster Platform Type
     ${odhdeployer_logs}=    Fetch Odh-deployer Pod Logs
     Should Contain    ${odhdeployer_logs}    ${odhdeployer_logs_content}
 
+Verify That The Operator Pod Does Not Get Stuck After Upgrade
+    [Documentation]    Verifies that the operator pod doesn't get stuck 
```

Comment:
I'm just wondering about the timing of this test. Would this run during or after the upgrade?

Reply:
During the upgrade, the test will work, but we need to know if a pod is stuck after the upgrade. So it's better after the upgrade.